packages/react-openapi/src/code-samples.ts

```diff
@@ -65,6 +65,24 @@ export const codeSampleGenerators: CodeSampleGenerator[] = [
             return lines.map((line, index) => (index > 0 ? indent(line, 2) : line)).join(separator);
         },
     },
+    {
+        id: 'python',
+        label: 'Python',
+        syntax: 'python',
+        generate: ({ method, url, headers, body }) => {
+            let code = 'import requests\n';
+            code += `response = requests.${method.toLowerCase()}("${url}",\n`;
```

Comment:
Should we add a line break before? Usually, it looks like Python has line breaks between imports and code body.